plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/portforwading/PortForwardingWizardModel.java

```diff
@@ -101,6 +106,7 @@ public IPortForwardable visit(final IPortForwardable cap) {
 		if(portForwardable != null) {
 			REGISTRY.put(pod, portForwardable);
 		}
+		firePropertyChange(PROPERTY_PORT_FORWARDING, false, getPortForwarding());
```

Comment:
@adietish I thought his pull said the property change event we fired in the code up a few lines didnt do anything and this was the reason the new line was added.  I would think we would remove the original line then